Orange/evaluation/testing.py

```diff
@@ -202,6 +202,53 @@ def get_fold(self, fold):
 
         return results
 
+    def get_table_with_probabilities(self, model_names):
+        """
+            Return the Orange.data.Table with data, predictions and probabilities (if the task is classification).
+
+            :param model_names: A list of strings containing learners' names.
```

Comment:
Would you consider writing any new docstrings in [Napoleon style](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/index.html)?